src/appFunctions.js

```diff
@@ -27,4 +27,18 @@ export function changeStatusCard(id, newStatus, {statuses, dataByStatuses}) {
         dataByStatuses: dataByStatusesNew,
         statuses
     });
+}
+
+export function sortCardsByTitle(cards, ascending) {
+    let cardsSort = cards.sort((a, b) => {
+        let charA = a.title.toUpperCase();
+        let charB = b.title.toUpperCase();
+
+        if (charA > charB) {
+            return 1;
+        }
+        return -1;
+    });
+
+    return ascending ? cardsSort.reverse() : cardsSort;
```

Comment:
not good, with `reverse` you will sort array 2 times, use ternary operator based on `ascending` flag